docs/reference/contrib/modeladmin/indexview.rst

```diff
@@ -617,7 +617,7 @@ kind of interactivity using JavaScript:
 ``wagtail.contrib.modeladmin.mixins.ThumbnailMixin``
 ----------------------------------------------------
 
-If you're using ``wagtailimages.Image`` to define an image for each item in
+If you're using ``media.Image`` to define an image for each item in
```

Comment:
Seeing as not all projects use the project template, I feel it might be better to say something like "If you're using `wagtailimages.Image` (or a custom Image model)" here instead.